packit_service/worker/handlers/github_handlers.py

```diff
@@ -612,6 +599,20 @@ def run(self) -> TaskResults:
             self.project.issue_comment(self.db_trigger.issue_id, msg)
             return TaskResults(success=False, details={"msg": "Propose update failed"})
 
+        local_project = LocalProject(
+            git_project=self.project,
+            working_dir=self.service_config.command_handler_work_dir,
+            ref=self.data.tag_name,
```

Comment:
in tests, I'd make sure that tag_name is properly parsed from the webhook and set in the Event

Reply:
Yeah, actually the failing test above needed just that to be fixed.